build.xml

```diff
@@ -8,50 +8,61 @@
 
     <!-- Target: install-all -->
     <target name="install-all" description="Install all three Drupal sites in the Federated Search Demo">
+        <phingcall target="solr-clear" />
         <phingcall target="install-d7" />
 
         <phingcall target="install-d8">
             <property name="drush.root" value="web/d8/docroot" override="true" />
+            <property name="site.name" value="Federated Search Drupal 8" override="true" />
         </phingcall>
-
+        <!-- This target has not been updated
         <phingcall target="install-d8">
             <property name="drush.root" value="web/d8-domain/docroot" override="true" />
-        </phingcall>
+            <property name="site.name" value="Federated Search Drupal 8 Domain" override="true" />
+        </phingcall>-->
     </target>
 
 
     <!-- Target: install-d8 -->
     <target name="install-d8" description="Install one of the Drupal 8 sites; specify which site with -Ddrush.root=web/d8/docroot">
         <property name="drush.root" value="web/d8/docroot" />
+        <property name="site.name" value="Federated Search Drupal 8" />
 
         <exec command="composer install" dir="${drush.root}/.." passthru="true" />
         <phingcall target="install-drupal-by-path" />
 
         <!-- Reinstall the Umami demo content -->
-        <drush command="pm-uninstall" assume="yes"><param>demo_umami_content</param></drush>
-        <drush command="pm-enable" assume="yes"><param>demo_umami_content</param></drush>
+        <!--drush command="pm-uninstall" assume="yes"><param>demo_umami_content</param></drush-->
+        <!--drush command="pm-enable" assume="yes"><param>demo_umami_content</param></drush-->
 
         <!-- Create demo content -->
         <drush command="pm-enable" assume="yes">
-            <param>federated_terms</param>
             <param>federated_default_content</param>
         </drush>
-
+        <!-- Copy demo files -->
+        <copy todir="web/d8/docroot/sites/default/files" mode="777">
+          <fileset dir="web/d8/docroot/modules/custom/federated_default_content/assets">
+            <include name="**/*.jpg" />
+          </fileset>
+        </copy>
         <!-- Uninstall the demo content modules -->
         <drush command="pm-uninstall" assume="yes">
-            <param>federated_terms</param>
             <param>federated_default_content</param>
         </drush>
 
         <drush command="search-api:clear" assume="yes">
             <param>federated_search</param>
         </drush>
-        <drush command="search-api:index" assume="yes">
-            <param>federated_search</param>
-        </drush>
 
-        <!-- Import config to fix rogue blocks changing UUIDs -->
-        <drush command="config-import" assume="yes" />
+        <exec command="drush @d8 search-api:index federated_search -y" />
+        <!-- If we run the above normally, we get the error:
```

Comment:
This is such a weird drush bug! I get the same error on the command line as well running `drush --root=web/d8/docroot search-api:index federated_search -y`, so it's not like it's a phing problem.

Reply:
Right. I think it's a "these chained commands fail" issue in running a batch operation that doesn't inherit vars.